examples/using-gatsby-image/plugins/gatsby-source-remote-images/gatsby-node.js

```diff
@@ -4,9 +4,11 @@ exports.onCreateNode = async (
   { actions: { createNode }, node, createContentDigest, store, cache },
   { filter, nodeName = `localFile` }
 ) => {
+  // asdf
   if (filter(node)) {
     const fileNode = await createRemoteFileNode({
       url: node.url,
+      parentNodeId: node.id,
```

Comment:
This is, actually pretty important 😱 Subsequent re-builds fail if we don't add this.